src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+import React from 'react';
+import { useLocation } from 'react-router';
+import ViewCount from '../../../../utils/ViewCount';
+import IcTwotoneDelete from '../../Icons/IcTwotoneDelete';
+import "./PlayListVideoCards.css";
+
+function PlayListVideoCards({props,setPlayListVideos}) {
+    const { videoid, snippet, statistics } = props;
+    const location = useLocation();
+    const playlistId = location.search.slice(1);
+    console.log(props, videoid, snippet, statistics,playlistId)
+
+    const deletePlayListVideoFromIindividualPlayListHandler = async(id) => { 
+        try {
+            console.log(id)
+            var res = await axios.delete(`/api/user/playlists/${playlistId}/${videoid}`,{ headers:{
+                authorization:localStorage.getItem("jafnaToken")}});
+                console.log(res);
+                setPlayListVideos(res.data.playlist.videos);
+            // setHistoryContextArray();
+            (async() => {
+                var res = await axios.get(`/api/user/playlists/${playlistId}`,{ headers:{
+                  authorization:localStorage.getItem("jafnaToken")}});
+                console.log(res);
+
+                // console.log(res.data.products);
+                // setHistoryContextArray(res.data.history);
+               
```

Comment:
Can you tell me why you are doing this?

Reply:
fetching details of a single playlist by id